src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -497,9 +504,23 @@ public override int Execute(DocumentsOperationContext context)
                     }
 
                     var document = documentType.Document;
-
                     var id = document.Id;
 
+                    if (documentType.Counters != null)
+                    {
+                        foreach (var counter in documentType.Counters)
+                        {
```

Comment:
Here we have to make an interesting decision. For documents, we don't keep the change vector after the import.
I think it would be better to have just `IncrementCounter` here, instead. With the total value.
The output that you generate should stay the same, though.

Reply:
Exporting and importing will give you a different result.
A: 1, B: 1 -> Export: A:2 -> Import: A:3, B:1